HW02_B_Mimic_starter - Viola graphs.Rmd

```diff
@@ -0,0 +1,178 @@
+---
+title: "HW02_B_Graph-Mimic"
+author: "Viola Nawrocka"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+
+```{r load libraries, echo = TRUE, include=TRUE}
+library("ggplot2")
+library("magrittr") #so I can do some piping
+data("diamonds")
+data("mpg")
+data("iris")
+theme_set(theme_bw()) #I'll give you this one, you can set the theme individually for graphs
+#or you can set the theme for all the graphs using theme_set()
+#theme_bw() is best theme (IMO)
+
+#for graph 3:
+library("ggrepel")
+
+
+```
+
+## HW02 Part B
+
+For this part of the HW, the goal is to try to recreate the graphs I make from scratch. I will only provide the MD, not the actual code I used to create it besides which data I use to create it. The rest will be up to you. 
+
+Try for all 4, but if you are struggling don't worry about it. Try your best for each, if you don't get everything that's what the peer-review is for. :smile:
+
+
+### Graph 1
+```{r, echo = T, include = TRUE}
+
+#hint think about the *position* the bars are in...
+```
+
+
+Using the diamonds dataset, make this graph:
+```{r graph1 code, echo=FALSE}
+
+data("diamonds")
+
+diamonds %>% 
+  ggplot(mapping = aes(x = cut, fill = clarity)) +
+  geom_bar(position = position_dodge(width = 0.9)) +
+  labs(
+      title = "My Diamond Collection",
+      subtitle = "Boxplot representing the number of diamonds in my diamond collection by \ntype of cut quality and clarity of diamond",
+      x = "Diamond Cut",
+      y = "Number of Diamonds"
+  ) +
+  theme(plot.title = element_text(hjust = 0.5)) +
+  annotate("text", x = "Premium", y = 4500, label = "My Best Diamonds, \n of course") +
+  annotate ("rect", xmin = 4.5, xmax = 5.5, ymin = 0, ymax = 5000, hjust = "Ideal", fill = "grey40", alpha = 0.3)
+
+```
+
+### Graph 2
+```{r, echo = T, include = TRUE}
+data("iris")
+
+```
+
+Using the iris dataset, make this graph:
+```{r graph 2 code, echo=FALSE}
+data("iris")
+
+arrange(iris, Species)
+
+iris %>% 
+  ggplot(mapping = aes(x = Sepal.Length, y = Petal.Length, color = Species)) +
+  geom_point() +
+  geom_smooth(method= lm, se = F, color = "black") +
+  facet_wrap(~ Species, scales = "free_y")
```

Comment:
Nice job finding facet_wrap, that took me a while and I don't think a lot of other people figure it out ;)